king_phisher/client/windows/plugin_manager.py

```diff
@@ -339,13 +350,39 @@ def _plugin_enable(self, model_row):
 		self._set_model_item(model_row.path, enabled=True, sensitive_installed=False)
 		self.config['plugins.enabled'].append(named_row.id)
 
+	def _compatibility_to_dict(self, compatibility_details):
+		required_packages = []
+		kp_version = None
+		python_version = None
+		supported_os = None
+
+		for detail in compatibility_details:
+			if detail[0].lower() == 'required package':
```

Comment:
Use the `RequirementType` enum you defined here. Should be able to simply do `detail == RequirementType.PYTHON_VERSION` etc.